src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+// @import "../utils/functions";
+// @import "../utils/mixins";
+// @import "../utils/extends";
```

Comment:
I guess it would be better to remove comments

Reply:
Agree, delete redundant comments